README.md

```diff
@@ -1,56 +1,60 @@
 
-```bash
-IMAGE_REGISTRY=<my registry>
-make image
-```
+## Getting started with the Node Feature Discovery Operator
 
-Update the `Makefile` and edit `IMAGE`, `ORG` and `REGISTRY` one will need those later to update the operator manifest (`image: $(REGISTRY)/$(ORG)/$(cluster-nfd-operator:$(TAG))`).
+> Prerequisite: a running OpenShift cluster 4.6+ 
 
-Update the Dockerfile with the correct golang build image and the base image the operator will run with.
-
-## Manual deploy of the operator
-Checkout the sources
+Get the source code
 
+```bash
+git clone -b {{ site.release }} https://github.com/openshift/cluster-nfd-operator
 ```
-$ git clone https://github.com/openshift/cluster-nfd-operator
 
-The default `NodeFeatureDiscovery` CR will create the operand (NFD) in the `node-feature-discovery-operator` namespace,
-also an empty [nfd-worker-conf](https://kubernetes-sigs.github.io/node-feature-discovery/v0.6/get-started/deployment-and-usage.html#configuration),
-the CR can be edited to choose another namespace, image, imagePullPolicy and nfd-worker-conf. See the `manifests/0700_cr.yaml` for the default values.
+Deploy the operator
 
 ```bash
-IMAGE_REGISTRY=<my registry>
+IMAGE_TAG={{ site.container_image }}
 make deploy
 ```
-Update the  `Makefile` with the a custom image built and configure the namespace where the operator should be deployed.
 
-The default CR will create the operand (NFD) in the `openshift-nfd` namespace, the CR can be edited to choose another namespace and image. See the `manifests/0700_cr.yaml` for the default values.
+Create a NodeFeatureDiscovery instance
 
+```bash
+oc apply -f config/samples/nfd.kubernetes.io_v1_nodefeaturediscovery.yaml
 ```
-$ cd cluster-nfd-operator
-$ make deploy
-```
-The operator will use the NFD image built from: https://github.com/openshift/node-feature-discovery
 
-To uninstall the operator run
-```
-$ make undeploy
-```
+## Verify
+
+The Operator will deploy NFD based on the information
+on the NodeFeatureDiscovery CR instance,
+after a moment you should be able to see
 
-To verify the correct working of NFD a e2e test can be run as well:
+```bash
+$ oc -n node-feature-discovery-operator get ds,deploy
+NAME                        DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/nfd-worker   3         3         3       3            3           <none>          5s
+NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/nfd-master   1/1     1            1           17s
 ```
-$ make test-e2e
+
+Check that NFD feature labels have been created
+
+```bash
+$ oc get no -o json | jq .items[].metadata.labels
```

Comment:
`oc get nodes`?

Reply:
I really like short names in kube, looks like you don't  jejeje , is ok

Reply:
`no` ;-)
but that's true I don't use it much, except `ds` I guess